docs/using.md

```diff
@@ -833,6 +834,58 @@ This will create new Actor .cpp and .h files, an example workload yaml, as well
 If your configuration wants to use logic, ifs, or anything beyond simple or existing commands in a loop, then consider writing your own Actor. It doesn't need to be super general or even super well-tested or refactored. Genny is open to submissions and you own whatever Actor you write. No need to loop TIPS in to your custom actor's PR unless you'd just like a second look.
 
 
+<a id="org627591d"></a>
+
+## Enabling Client-Side Field Level Encryption
+
+To enable encryption in your workload using either Field Level Encryption (FLE) or Queryable Encryption (QE), the schema for the encrypted collection must first be defined in the `Encryption.EncryptedCollections` node of the workload YAML file. The following example shows a FLE schema defined for namespace `encrypted_db.fle_encrypted_coll`, and a QE schema defined for namespace `encrypted_db.qe_encrypted_coll`:
+
+```yaml
+Encryption:
+  EncryptedCollections:
+  - Database: encrypted_db
+    Collection: fle_encrypted_coll
+    EncryptionType: fle
+    FLEEncryptedFields:
+      name: { type: "string", algorithm: "random" }
+      pii.ssn: { type: "string", algorithm: "deterministic" }
+  - Database: encrypted_db
+    Collection: qe_encrypted_coll
+    EncryptionType: qe
+    QueryableEncryptedFields:
+      name: { type: "string", queries: [{queryType: "equality"}] }
+      pii.ssn: { type: "string", queries: [{queryType: "equality"}] }
+```
+
+The required `Database` and `Collection` fields specify the namespace of the encrypted collection. Each node in this sequence must have a unique combination of these two names in order to avoid two different encryption schemas for the same namespace on the same URI.
```

Comment:
Should we raise a feature request to have genny enforce this?

Reply:
This is already enforced in [here](https://github.com/mongodb/genny/blob/06c2e78ae75850fc9bc73922a87e650a88bfb84d/src/gennylib/src/encryption.cpp#L617:L623)